src/Settings/MatrixViewSettings.ts

```diff
@@ -188,5 +188,6 @@ export function addMatrixViewSettings(
         await plugin.saveSettings();
         await plugin.getActiveTYPEView(MATRIX_VIEW).draw();
       })
-    }
+    })
```

Comment:
This change means it either introduces a syntax error, or there was already a syntax error... right?

Reply:
yes it is a syntax error